api-common-java/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,51 +0,0 @@
----
-name: Bug report
-about: Create a report to help us improve
-
----
-
-Thanks for stopping by to let us know something could be better!
-
-**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
```

Comment:
Should we have issue templates in the monorepo like this one?

Reply:
Added with a few edits to reference this monorepo.